Eastern-Research-Group/csb-rebate-forms-app
# Update 2024 NCES JSON

## Related Issues:
* CSBAPP-444

## Main Changes:
* Yet another follow-up to #490 – Update 2024 NCES JSON

## Steps To Test:
1. Navigate to a 2024 FRF submission, and advance to the "School District Info" page
2. Type in an NCES District ID for a school district that has been removed (e.g. 3600076) and ensure using it no longer returns a result.
